pkg/deploy/devconfig.go

```diff
@@ -171,6 +172,13 @@ func DevConfig(_env env.Core) (*Config, error) {
 			MDMFrontendURL:              pointerutils.ToPtr("https://global.ppe.microsoftmetrics.com/"),
 			MDSDEnvironment:             pointerutils.ToPtr(version.DevGenevaLoggingEnvironment),
 			MsiRpEndpoint:               pointerutils.ToPtr("https://iamaplaceholder.com"),
+			MiseValidAudiences: []string{
+				"https://management.core.windows.net/",
+				_env.Environment().ResourceManagerEndpoint,
+			},
+			MiseValidAppIDs: []string{
+				"2187cde1-7e28-4645-9104-19edfa500053",
```

Comment:
Can we make a more descriptive variable for this? Who's app id is this?

Reply:
+1, could we make this a configuration variable?

Reply:
Variable name `MiseValidAppIDs` signifies list of Azure AD App IDs which are authorised to send request for authentication to MISE. As of now the app id is of ARO-RP but it can contain any other app id for a service we want to authenticate for. So keeping the variable name in reference to MISE rather than to the services whose app id will be added as a valid app id.

Reply:
Could you add a comment in the code to describe this to make it more readable, where the id comes from, what it stands for .etc.

Reply:
The id used here is a dummy id for dev. The actual list of IDs comes from our repository which maintains configurations for RP. However, I can include a comment describing what this variable stands for.